talent/onboarding/Non-EU_residents.md

```diff
@@ -88,4 +88,41 @@ If the candidate has a family, he needs to [ask for an appointment](https://w6.s
 
 With this document the candidate has to go to the closest clinic (Centro de Salud) to their house and ask for the Health card (Tarjeta sanitaria).
 
+## Spanish Residence Permit Renewal 
+
+We have to apply to renew the resident card within 60 days before the card expires. See below a list of documents we will need to renew the permit. 
+
+1. Company:
+   - Informe de Comercio
+
+2. Employee:
+   - Passport copy - all pages. We need to show them you didn’t leave the country more than 6 months per year.
+   - Resident card - copy
+   - Contract work - copy
+   - Last three payslips - copy
+   - Letter from HR department with a certification of the employee is working at the same position.
```

Comment:
@estherrgarcia does the person need to be working in the *very same* position? unlikely, no?

Reply:
This person came to work here because it was the best person to do a specific work here, we explained why when we applied for the high qualification Visa. I guess, for this reason, the Government wants to know it.

Reply:
Well, I changed job titles… 🤔

Reply:
We will review every case when we have to! No worries! We will handle it :)